release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -995,6 +1000,11 @@ Your upgrade to {product-title} 4.6 should now no longer be blocked by this feat
 
 The Node Maintenance Operator now validates maintenance requests for master nodes, preventing master (etcd) quorum violation. As a result, master nodes can only be set into maintenance if the `etcd-quorum-guard` pod disruption budget (PDB) allows it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826914[*BZ#1826914*])
 
+[id="ocp-4-6-noe-maintenance-operator-cr"]
```

Comment:
probably not important, but there is a typo in the id, missing `d` in `node`

Reply:
@slintes fixed anyways, thanks.